### PR TITLE
chore: update .gitignore

### PR DESCRIPTION
**Pull request by Yaronglp**
Avoid developers from publicly exposing their personal key

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,7 @@ yarn-error.log*
 # env files
 .env
 .env.local
+.env.example
```
Comment (tanvirstreame) [inline .gitignore:29]: This should not be in gitignore
This is for the developer to know what data should be in the .env file.

Comment (Yaronglp) [reply .gitignore:29]: Hi @tanvirstreame ,

The file will be visible, but new users will be prevented from accidentally pushing their keys.

Comment (tanvirstreame) [reply .gitignore:29]: Hey @Yaronglp This `env.example` is for giving ideas about what should be in `env`. Also, keeping the value in `env.example` will not affect anything in the application.

Comment (schnerd) [reply .gitignore:29]: agree with @tanvirstreame

**Pull request closed without merging**